pkg/util/tmpdir/tmpdir.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package tmpdir provides helper getting a temporary directory for tests that
+// write to the filesystem.
+package tmpdir
```

Comment:
As this package is used only for tests, should we create a `test_utils` folder instead and add the package in there instead of adding to `utils`?